src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-day--sun calendar--month-length--30">
+        <div class="square"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
```
it should be bem element